staticd/static_main.c

```diff
@@ -0,0 +1,151 @@
+/*
+ * STATICd - main code
+ * Copyright (C) 2018 Cumulus Networks, Inc.
+ *               Donald Sharp
+ *
+ * FRR is free software; you can redistribute it and/or modify it
```

Comment:
Please use the header given in the dev docs.
http://frrouting.readthedocs.io/projects/dev-guide/en/latest/workflow.html#source-file-header

Reply:
Fixed.